great_expectations/rule_based_profiler/parameter_builder/regex_pattern_string_parameter_builder.py

```diff
@@ -234,63 +234,27 @@ def _build_parameters(
             parameters=parameters,
         )
 
-        # get list of regex_strings that match greater than threshold
-        regex_string_success_list: List[
-            str
-        ] = self._get_regex_matched_greater_than_threshold(
+        # get best-matching regex_string that match greater than threshold
+        (
+            best_regex_string,
+            best_ratio,
+        ) = ParameterBuilder._get_best_candidate_above_threshold(
             regex_string_success_ratios, threshold
         )
         # sorted regex and ratios for all evaluated candidates
-        sorted_ratio_list, sorted_regex_string_list = self._get_sorted_regex_and_ratios(
+        (
+            sorted_ratio_list,
+            sorted_regex_string_list,
+        ) = ParameterBuilder._get_sorted_candidates_and_ratios(
             regex_string_success_ratios
         )
 
         return (
-            regex_string_success_list,
+            best_regex_string,
             {
+                "success_ratio": best_ratio,
                 "evaluated_regexes": dict(
                     zip(sorted_regex_string_list, sorted_ratio_list)
                 ),
-                "threshold": threshold,
```

Comment:
@Shinnnyshinshin Yes, this looks correct -- just the way of computing `sorted_regex_string_list` and `sorted_ratio_list` can be made simpler / more efficient.  Thanks.